apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -2971,6 +2971,7 @@ const applyMarketplaceModelEpic: AppEpic = (action$, state$) =>
           );
 
         return concat(
+          of(MarketplaceActions.setDetailsModel()),
```

Comment:
maybe it's better to add it to `useEffect` `return` statement? To prevent unexpected behaviour if we somehow go back to Chat without clicking on use model button?

Reply:
we can't go to the chat without closing app card